### ISSUE TITLE
Search Editor: Add keybinding for double-click action, e.g. to open to the side

### ISSUE THREAD

**Issue by jdanbrown** [microsoft/vscode#194044]
In the Search Editor, there are currently keybindings / keyboard shortcuts for opening results in the current editor group, and double-clicking can be configured to open to the side, but there are currently no keybindings for opening to the side:

- https://code.visualstudio.com/docs/editor/codebasics#_search-editor
> In the Search Editor, results can be navigated to using Go to Definition actions, such as F12 to open the source location in the current editor group, or ⌘K F12 to open the location in an editor to the side. Additionally, double-clicking can optionally open the source location, configurable with the search.searchEditor.doubleClickBehaviour setting.

- https://code.visualstudio.com/docs/getstarted/settings
```js
  // Configure effect of double-clicking a result in a search editor.
  //  - selectWord: Double-clicking selects the word under the cursor.
  //  - goToLocation: Double-clicking opens the result in the active editor group.
  //  - openLocationToSide: Double-clicking opens the result in the editor group to the side, creating one if it does not yet exist.
  "search.searchEditor.doubleClickBehaviour": "goToLocation",
```

It'd be great to add a keybinding to do the same action as double-click, which users can already configure to `openLocationToSide`. (Looks like `enter` / `cmd+enter` are currently available.)

**Comment by weartist** [comment]
@andreamah I'd be willing to fix this if I could

**Comment by ShiviTripathi13** [comment]
@andreamah if the issue currently open, please assign

**Comment by andreamah** [comment]
> @andreamah I'd be willing to fix this if I could

Yes, sure! I think you can just abstract https://github.com/microsoft/vscode/blob/7906b2bb614754f39a12bb4aa2d143e637f47694/src/vs/workbench/contrib/searchEditor/browser/searchEditor.ts#L249-L260 into its own function and make a search action that also calls it from a keybinding (with `searchEditor.action.*` as an action ID).

**Comment by weartist** [comment]
> > @andreamah I'd be willing to fix this if I could
> 
> Yes, sure! I think you can just abstract
> 
> https://github.com/microsoft/vscode/blob/7906b2bb614754f39a12bb4aa2d143e637f47694/src/vs/workbench/contrib/searchEditor/browser/searchEditor.ts#L249-L260
> 
> into its own function and make a search action that also calls it from a keybinding (with `searchEditor.action.*` as an action ID).

Thank you for the tip~I'll fix it

**Comment by DatN99** [comment]
@weartist If you're not working on this issue anymore, I'd love to give it a shot!

**Comment by weartist** [comment]
> @weartist If you're not working on this issue anymore, I'd love to give it a shot!

Yes, I didn't continue looking into this anymore
